packages/core/src/domain/session/sessionManager.ts

```diff
@@ -41,8 +44,10 @@ export function startSessionManager<TrackingType extends string>(
 
   sessionStore.renewObservable.subscribe(() => {
     sessionContextHistory.add(buildSessionContext(), relativeNow())
+    renewObservable.notify()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding some tests to express the relationship between observables and contexts?

Reply:
https://github.com/DataDog/browser-sdk/blob/b1733892001f9e2baf0e5ee75b7228f6dc3d81bb/packages/core/src/domain/session/sessionManager.spec.ts#L501-L524